### DISCUSSION TITLE
posts/fast-log2

### DISCUSSION THREAD

**Discussion by giscus** [EarthMessenger/blog.earthmessenger.xyz#3] [giscus]
# 2022/01/11/fast-log2

众所周知, \(\lg\) 是 OI 中十分常用的操作。

https://earthmessenger.github.io/2022/01/11/fast-log2.html

**Comment by robinyqc** [comment]
补充一句：g++ 的 std::__lg 函数同样使用 __builtin_clz，实现完全一样，可以作为文中写法的简单替代。详情可见源码。
